tests/e2e/tidbcluster/serial.go

```diff
@@ -747,7 +748,32 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			framework.ExpectNoError(err, "create pdapi error")
 			defer cancel()
 			var firstScaleTimestamp int64
-			err = wait.Poll(10*time.Second, 10*time.Minute, func() (done bool, err error) {
+			var readyToScaleTimestamp int64
+			err = wait.Poll(10*time.Second, 5*time.Minute, func() (done bool, err error) {
+				tac, err = cli.PingcapV1alpha1().TidbClusterAutoScalers(ns).Get(tac.Name, metav1.GetOptions{})
+				if err != nil {
+					return false, nil
+				}
+				if tac.Annotations == nil || len(tac.Annotations) < 1 {
+					klog.Infof("tc haven't marked any annotation")
```

Comment:
use `tac haven't marked any annotation` instead and use `framework.Logf` to record the log.